src/server.cpp

```diff
@@ -127,9 +127,13 @@ void *ServerThread::run()
 
 		u64 t0 = porting::getTimeUs();
 
-		const Server::StepSettings step_settings = m_server->getStepSettings();
+		const auto step_settings = m_server->getStepSettings();
 
 		try {
+			// see explanation inside
+			if (dtime > step_settings.steplen)
+				m_server->yieldToOtherThreads(dtime);
```

Comment:
i think we should say dispatch instead of yield

Reply:
yield is the correct term: https://man7.org/linux/man-pages/man2/sched_yield.2.html